### PR TITLE
Whats new improvements and additions of missing items.

### PR DESCRIPTION
**Pull request by lbdreyer**

### CODE REVIEW

File: docs/iris/src/whatsnew/1.12.rst [hunk 1]
```diff
@@ -102,9 +104,20 @@ Iris 1.12 Features
...
 
-* The transpose method of a Cube now results in a lazy transposed view of the original rather than realising the data then transposing it.
+* The transpose method of a Cube now results in a lazy transposed view of the
+  original rather than realising the data then transposing it.
+
+* The :func:`iris.analysis.cartography.area_weights` function is now more
+  accurate for single precision input bounds.
+
+* Iris is now able to read seconds in datetimes provided in NAME trajectory
+  files.
```
Comment (lbdreyer) [inline docs/iris/src/whatsnew/1.12.rst]: #2266

File: docs/iris/src/whatsnew/1.12.rst [hunk 2]
```diff
@@ -102,9 +104,20 @@ Iris 1.12 Features
...
+
+* Optimisations to trajectory interpolations have resulted in a significant 
+  speed improvement.
```
Comment (lbdreyer) [inline docs/iris/src/whatsnew/1.12.rst]: #2252

File: docs/iris/src/whatsnew/1.12.rst [hunk 3]
```diff
@@ -102,9 +104,20 @@ Iris 1.12 Features
...
+  
+* Many new and updated translations between CF spec and STASH codes.
```
Comment (lbdreyer) [inline docs/iris/src/whatsnew/1.12.rst]: #2298

File: docs/iris/src/whatsnew/1.12.rst [hunk 4]
```diff
@@ -114,4 +127,8 @@ Deprecations
 
 Documentation Changes
 =====================
-* Corrected documentation of :class:`iris.analysis.AreaWeighted` scheme to make the usage scope clearer.
+* Corrected documentation of :class:`iris.analysis.AreaWeighted` scheme to make
+  the usage scope clearer.
+  
+* A chapter on :doc:`the new graphical testing technique </developers_guide/graphics_tests>`
+  has been added to the :doc:`developers guide </developers_guide/index>`.
```
Comment (lbdreyer) [inline docs/iris/src/whatsnew/1.12.rst]: #2294, #2185

**Comment by bjlittle** [comment]
@lbdreyer Awesome, thanks :+1:

**Pull request merged**